frontend/public/index.html

```diff
@@ -17,6 +17,7 @@
       href="https://use.fontawesome.com/releases/v5.0.13/css/all.css"
       rel="stylesheet"
     />
+    <link href="https://cdn.materialdesignicons.com/4.0.96/css/materialdesignicons.min.css" rel="stylesheet">
```

Comment:
Can't we find all the icons in one single icon set? I wouldn't like us to use two sets.